internal/commands/pull/command.go

```diff
@@ -168,30 +195,39 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 	ui.Print(terminal.NewTextLog("Saved app to disk"))
 
-	if cmd.inputs.IncludeDependencies {
+	if cmd.inputs.IncludePackageJSON || cmd.inputs.IncludeNodeModules {
```

Comment:
```suggestion
	if cmd.inputs.IncludePackageJSON || cmd.inputs.IncludeNodeModules || cmd.inputs.IncludeDependencies {
```

---

also, a totally minor/optional/ocd thing, but it seems most everywhere else you have `NodeModules` checked prior to `PackageJSON`.  consistency there would certainly help the codebase's grep-ability